### DISCUSSION TITLE
Help with creating and viewing new projects on Quire (Windows 11)

### DISCUSSION THREAD

**Discussion by miandfetter** [thegetty/quire#935] [Get Help] [answered]
Hello all,
I'm new to Quire and I'm having difficulty with the installation and running basic commands on the powershell to create and view my new projects. I followed the steps to download node.js, git, and set my execution policy to unrestricted, but I'm still getting this error when creating a new project:

![quire_new_error](https://github.com/thegetty/quire/assets/84156113/a86bc4a0-77e9-4fb3-886d-72adf2d4c806)

PS C:\Users\mndousse> quire new testing2
[CLI:quire] init-starter
  project root: "testing2"
  starter: "https://github.com/thegetty/quire-starter-default"
testing2 set to use quire-11ty@1.0.0-rc.16
[CLI:quire] installing @thegetty/quire-11ty@1.0.0-rc.16 into testing2
Error encountered while installing @thegetty/quire-11ty@1.0.0-rc.16 NPM package.
Error: ENOENT: no such file or directory, mkdir 'C:\Users\mndousse\testing2\node_modules\.install-npm-version-temp-n7MwIk\node_modules\@thegetty\quire-11ty\node_modules'

The projects are created in my directory (testing2 does show up for ls) but the preview command doesn't work as well. Any tips would be much appreciated, thanks!

**Comment by Erin-Cecele** [comment]
Thank you @miandfetter. Sorry to hear you are encountering issues. Can you run `node --version` and let me know what version of Node.js you are running?

**Reply by miandfetter** [reply]
Sure! It says I'm using Node v20.14.0.

**Comment by Erin-Cecele** [comment]
Hi @miandfetter it may be your newer version of Node that is causing difficulties. Try installing Node v. 18.20.0 via this link: https://nodejs.org/download/release/v18.20.0/ (`win` indicates the Windows versions.) Confirm you are running the correct version of Node by running `node --version` and then try `quire new` again. Also don't use `testing2` again. Trying a different project name. Let us know if that works!

**Reply by Erin-Cecele** [reply]
@miandfetter I saw you commented that you were having trouble getting your path to recognize version 18 of Node.js. However, that comment is no longer appearing in this thread. Were you able to get it working? If not, check out this article related to Node version management for Windows: https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows.

**Reply by miandfetter** [reply]
@Erin-Cecele Thank you so much for following up! I found some resources on nvm and tried to use version 18.20.0, but I then checked the version my system was running, and it still said 20.14.0. I think the newer version of node is overriding the older version, but I can't seem to be able to remove it from my system. I've tried deleting node from my apps, the control panel, and file explorer but node -v still returns the newer version.

**Reply by Erin-Cecele** [reply]
Hi @miandfetter. Check out this resource: https://phoenixnap.com/kb/downgrade-node-version-windows. Is this something you have tried yet?

**Reply by miandfetter** [reply]
Yes @Erin-Cecele, I tried this method that the resource you shared explains, but the newer version (20.14.0) still shows up. I'm trying to find the 20.14.0 version to uninstall it and hopefully my computer will only use the 18.20.0 version.

**Reply by miandfetter** [reply]
Also, @Erin-Cecele I'm now able to use Quire because I switched to a mac. Thank you so much for your responses and helping me the past several days!

**Reply by Erin-Cecele** [reply, marked as answer]
Thanks for the updates @miandfetter. Glad to hear Quire is working for you now! What version of Node.js are you using on the Mac?

**Reply by miandfetter** [reply]
I'm using version 18.20.4 (LTS)
